libwebauthn/src/transport/hid/device.rs

```diff
@@ -102,23 +91,3 @@ impl<'d> Device<'d, Hid, HidChannel<'d>> for HidDevice {
     //     channel.supported_protocols().await
     // }
 }
-
-#[cfg(test)]
-mod tests {
-
-    #[cfg(feature = "hid-device-tests")]
```

Comment:
I agree with removing this feature as we only had one test, but I would love to have some E2E tests run automatically on `cargo test`.

I think we would want tests to be as comprehensive as possible, whilst examples would merely need to be readable/understandable - so they serve different purposes. Something we can add in the future?

Reply:
Now examples are not using virtual devices at all (because cargo won't allow me to do that, sadly), and all E2E-tests are run upon `cargo test`. Also in the CI.